civihr_default_theme/template.php

```diff
@@ -355,3 +355,63 @@ class="col-sm-3 control-label ' . ( $label_hidden ? 'hidden-xs' : '' ) . '">'
   return $fields_structure;
 }
 
+/**
+ * Returns HTML for a menu link and submenu.
+ *
```

Comment:
You are doing a lot of stuff in this method, please explain what in the documentation